### ISSUE TITLE
battery module time to empty

### ISSUE THREAD

**Issue by loonymike** [tobi-wan-kenobi/bumblebee-status#296]
#### Support Question

I have power installed via pip but the module still will not display time to empty. Power 1.4 Python 3.6.5 Arch Linux kernel 4.18.7

Thanks.

**Comment by fredj** [comment]
Hello,
In the `/sys/class/power_supply` directory you should have one or more directories starting with `BAT`. Can you add a comment with the content of theses directories?

**Comment by fredj** [comment]
can you run bumblebee in [debug mode](https://github.com/tobi-wan-kenobi/bumblebee-status#errors) and only include the battery module? something like:
```
$ ./bumblebee-status -d -m battery
```

**Comment by loonymike** [comment]
{"version": 1, "click_events": true}
[
[{"full_text": " 73% (03:39h) dis ", "color": "#aabbcc", "background": null, "separator_block_width": null, "separator": true, "min_width": "100%AAAAAA", "align": null, "instance": "f6d53504-7d26-46bf-8694-b43dd55b0312", "name": "battery"}],

**Comment by tobi-wan-kenobi** [comment]
Hi @loonymike,

in the debug output, the estimated time remaining seems to be present, am I correct (the 03:39)?

Can you please paste the full invocation line for bumblebee-status in your i3 config? There's a switch that indicates whether the remaining time should be displayed or not, maybe that is the issue...

**Comment by loonymike** [comment]
status_command  ~/github/bumblebee-status/bumblebee-status -m load sensors nic battery date time -p interval=2 time.format="%H:%M" date.format="%a, %b %d" home.path=/home -t iceberg-powerline

**Comment by loonymike** [comment]
![2018-09-15-025015_1280x800_scrot](https://user-images.githubusercontent.com/30101473/45585010-22009680-b892-11e8-9341-741136031f7c.png)

**Comment by tobi-wan-kenobi** [comment]
I agree with what @fredj wrote, that is the most likely explanation, I think. @fredj impressive that you remembered the #234 :+1:

**Comment by loonymike** [comment]
OK, I'll check it out I do use Anaconda. Thanks.
